pkg/platform/platform.go

```diff
@@ -74,6 +74,13 @@ type Platform interface {
 	Meta() Meta
```

Comment:
Some nits on the commit message. You used `interface` twice at the end of line 3. Also comma on line 3. I think it should be `period`.

Reply:
I _think_ I fixed it :smile: Please have a look again.